### ISSUE TITLE
1.19 WITHOUT building the gradle

### ISSUE THREAD

**Issue by DiscoDude96** [MrScautHD/Beyond-Earth#156]
### Describe the feature

I do not understand why the fuck this has to be so complicated, I am literally breaking into tears, it's time somebody does something about this. You do know there are uneducated beings on this platform that play 1.19 that actually have zero fucking clue on how to do this. I've asked nicely multiple times, but this has crossed the line for me.
Also my little bro got mad at this and smashed the keyboard so this is the new one, thanks alot.
I still don't know if this is actually being worked on, but if it's not, please just add the pre installed .jar file so I (ahem...) we don't lose the shit.

**Comment by SussyAmgogus** [comment]
Hey! :(
This is a duplicate of MY template!

**Comment by MrScautHD** [comment]
Pls, if you not know to build projects then just wait on the release version :)

**Comment by SussyAmgogus** [comment]
Yeah ikr like my guy needs to CHILL

**Comment by MrScautHD** [comment]
or check the dc for dev versions

**Comment by SussyAmgogus** [comment]
what's that? I'm new sorry
